<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_lesson_db</class>
 <widget class="QMainWindow" name="admin_lesson_db">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AtQlas Alpha</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="result_table">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>391</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PT Mono</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget 
{background-color: transparent;}
QScrollBar:vertical {
    background: url(images/pure_blue.png);
    width: 0px;
    margin: 0px 0 0px 0;
}
QScrollBar::handle:vertical {
    background: rgb(195, 195, 195);
    min-height: 30px;
    margin: 0 1px 0 2px;
    border-radius: 3px;
    border: none;
}

QScrollBar:horizontal {
    height: 0px;
    margin: 0px 0 0px 0;
}
QScrollBar::handle:horizontal {
    background: rgb(195, 195, 195);
    min-width: 20px;
    margin: 2px 0px 1px 0px;
    border-radius: 3px;
    border: none;
}
QHeaderView::section
{
border-image:url(:/atqres_ico/pure_blue.png);
height:30px;
spacing: 20px;
background-color:rgb(39,49,58);
color: white;

text-align: right;
font-family: arial;
font-size:12px;
}</string>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>150</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>45</red>
          <green>92</green>
          <blue>96</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>45</red>
          <green>92</green>
          <blue>96</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>45</red>
          <green>92</green>
          <blue>96</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>45</red>
          <green>92</green>
          <blue>96</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>45</red>
          <green>92</green>
          <blue>96</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>45</red>
          <green>92</green>
          <blue>96</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>45</red>
          <green>92</green>
          <blue>96</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>45</red>
          <green>92</green>
          <blue>96</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>45</red>
          <green>92</green>
          <blue>96</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>PT Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(45, 92, 96)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>nameLabel</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image:url(:/atqres_ico/blue_land.png)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="new_button">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/deep_plus.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_button">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/deep_minus.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="list_all">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/deep_circle.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="search_edit">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/pure_deep_water.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>800</width>
       <height>6</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/pure_deep_water.png)</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>35</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/deep_back.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>result_table</zorder>
   <zorder>name_label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
